app/router.go

```diff
@@ -71,7 +72,19 @@ func Router(c collector) *mux.Router {
 func makeReportPostHandler(a xfer.Adder) http.HandlerFunc {
 	return func(w http.ResponseWriter, r *http.Request) {
 		var rpt report.Report
-		if err := gob.NewDecoder(r.Body).Decode(&rpt); err != nil {
+
+		reader := r.Body
```

Comment:
It seems that the content in a POST is not automatically decompressed, even when the `Content-Type` is `gzip`... :worried: